### ISSUE TITLE
README.rst: Update the FAQ Link

### ISSUE THREAD

**Issue by shashank-b** [coala/coala-bears#2821]
The link in the Installation section of the file README.rst can be updated to the latest link:
http://docs.coala.io/en/latest/Help/FAQ.html
 
This can be a suitable issue for newcomers.

**Comment by ajupazhamayil** [comment]
Updated README.rst, Please approve my PR.

**Comment by naveentvelu** [comment]
@ajupazhamayil You are not allowed to make PR's unless you are assigned to it.
It is against the Community Guidelines.

**Comment by ajupazhamayil** [comment]
I am sorry that i did not have information about that, So should I close this PR?

**Comment by naveentvelu** [comment]
No, wait for the coala developers to look into this. I am also new here :)

**Comment by ajupazhamayil** [comment]
Thank you for your reply. I will wait then :)

**Comment by ajupazhamayil** [comment]
> No, wait for the coala developers to look into this. I am also new here :)
> Updated:
> Ask for the invitation in the gitter channel. And after receiving the invitation, ask any maintainers to assign the issue you want to work on.

Thank you for your reply. I will do in that way!

**Comment by utkarsh2102** [comment]
What?
I don't get the reason, @shashank-b.
Why should it be done? Isn't it updated already?

**Comment by rohanrajpal** [comment]
Hi! Can I be assigned this issue?

**Comment by tanvinerkar** [comment]
Hey! Can I be assigned this issue? Do let me know so I could start working on it. Thanks!

**Comment by divyanshu132** [comment]
@gitmate-bot is this issue still open?? can I take this!!

**Comment by nishant-sachdeva** [comment]
Is this issue closed or someone still working on it?
